apps/mobile/src/hooks/useHandleCurrentGroup.js

```diff
@@ -51,6 +52,9 @@ export function useHandleCurrentGroup () {
   const [{ currentGroup, fetching: fetchingCurrentGroup }] = useCurrentGroup()
   const loading = fetchingCurrentUser && fetchingCurrentGroup
 
+  const { context, groupSlug, originalLinkingPath } = useRouteParams()
+  const groupSlugFromPath = originalLinkingPath?.match(/\/groups\/([^\/]+)(?:\/|$)/)?.[1] ?? null
```

Comment:
In my mobile tracks branch, I have created a little pathHelper in /packages to check if there are tracks related params in the originalLinkingPath. Once that is merged across, I would move these little regexes into there